atlas_commons/utils.py

```diff
@@ -83,6 +83,27 @@ def split_into_halves(
     return left_volume, right_volume
 
 
+def assign_hemispheres(annotation_path, halfway_offset: int = 0, hemispheres_path = None):
+    annotation = voxcell.VoxelData.load_nrrd(annotation_path)
+
+    hemispheres_volume = np.zeros_like(annotation.raw, dtype=np.dtype('u1'))
+    z_halfway = hemispheres_volume.shape[2] // 2 + halfway_offset
```

Comment:
why not pass the halfway point, instead of an offset on that?  I think it would be clearer.

Reply:
I just used the same signature of the [split_into_halves function](https://github.com/BlueBrain/atlas-commons/blob/main/atlas_commons/utils.py#L64). Shall I pass the halfway point here instead?

Reply:
I added a [function](https://github.com/BlueBrain/atlas-commons/pull/4/commits/f2666d7dfcb63712b93250483931cd60ac84ca34#diff-0cf0dfd72ba1378805b48691b70030a99798daac57b3a480f07c143a3cc3c5c1R62) as it is used also in another place.

Reply:
sigh, too bad `split_into_halves` has that signature; I guess consistency is better.